lib/chef/application/client.rb

```diff
@@ -17,19 +17,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-require "chef/application"
-require "chef/client"
-require "chef/config"
-require "chef/daemon"
-require "chef/log"
-require "chef/config_fetcher"
-require "chef/handler/error_report"
-require "chef/workstation_config_loader"
-require "chef/mixin/shell_out"
+require_relative ""
```

Comment:
Does this just do the current dir?

Reply:
WTF?  no, that's a bug.

Reply:
weird that all those mangled lines didn't cause any test failures...